app/controllers/sessions_controller.rb

```diff
@@ -6,7 +6,7 @@ def create
     user = find_user
```

Comment:
either move this to a `before_action` or write a better function name and call it directly without assigning to the `user` variable.

{Remind to touch on this}